SU-HSDO/suhumsci
# feat(STN-414): 17.2 QA Feedback

# READY FOR REVIEW

## Summary
QA feedback from Sprint 17 (part 2), thus 17.2:

**Remaining TO-dos:**
- ~~[ ] table pattern does not work with responsive tables classes~~ This will not work with the table row pattern because of where the field classes are applied.
- [x] Utility classes for images on date stacked horizontal cards aren’t working as expected

**Browser Testing:**

## Steps to Test
1. Ensure that `npm run test` passes
2. Check that adding `hb-stretch-vertical-linked-cards` to the Components section of layout builder, no longer breaks:

- [x] vertical postcards with a long title http://economics.suhumsci.loc/qa/qa-cards
- [x] also, check that vertical linked postcards still retain the equal height behavior they had before: http://economics.suhumsci.loc/qa/qa-cards
   - I did find a very small bug that I was trying to resolve with the timeout function I added in the code, as well as trying to get an async function to work which neither I could quite get working... but basically the issue is if you are already on the QA-cards page and then click to go to that same page again, sometimes the card heights dont load properly and it will not always be even heights.
  - I should create a bug card for this
- [x] full width row wells with text inside http://economics.suhumsci.loc/qa/row-wells
- [x] full width row wells with accordions (bottom of page): http://economics.suhumsci.loc/qa/qa-page
3. Check that if a vertical linked card (only for Colorful theme) appears under the main nav dropdown, that it does not have a zindex anymore that makes it appear above the nav
- you may have to add vertical linked cards to a page to do this.
4. Check that mixing ordered and unorderered lists in sub list items in the WYSIWYG no longer results in odd appearance of ordered lists and their "." after the number anymore
5. All card image defaults (without util classes) should look unchanged:
    - [x] Horizontal Card
    - [x] Vertical Card
    - [x] Date Stacked Horizontal Card
    - [x] Date Stacked Vertical Card
    - [x] Vertical Linked Card
    - [x] Structured Card
5. All cards should work with `hb-card-image-` util classes:
    - [x] Horizontal Card
    - [x] Vertical Card
    - [x] Date Stacked Horizontal Card
    - [x] Date Stacked Vertical Card
    - [x] Structured Card

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
